supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/Delocalization.scala

```diff
@@ -95,12 +96,14 @@ trait Delocalization {
         |  while IFS= read line
         |  do
         |    gcs_path=$$(echo $$line | $prefixFilters)
-        |    ${recoverRequesterPaysError(cloudCallRoot)(gsutilCommand)}
+        |    (
+        |       ${retry(recoverRequesterPaysError(cloudCallRoot)(gsutilCommand))}
+        |    )
```

Comment:
This also fixes cron or any other cwl with more than one file to delocalize from the `cwl.output.json`